UnitTests/Onboarding/ContextualOnboarding/ContextualOnboardingViewHighlighterTests.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  ContextualOnboardingViewHighlighterTests.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+@testable import DuckDuckGo_Privacy_Browser
+
+final class ContextualOnboardingViewHighlighterTests: XCTestCase {
+
+    func testWhenHighlightViewIsCalledThenViewShouldContainAnimationView() {
+        // GIVEN
+        let (parent, child) = makeDummyViews()
+
+        // WHEN
+        ContextualOnboardingViewHighlighter.highlight(view: child, inParent: parent)
+
+        // THEN
+        XCTAssertNotNil(findFirstAnimationView(in: parent))
+    }
+
+    func testWhenIsViewIsHighligthedIsCalledAndViewIsHighlightedThenReturnTrue() {
+        // GIVEN
+        let (parent, child) = makeDummyViews()
+        XCTAssertFalse(ContextualOnboardingViewHighlighter.isViewHighlighted(child))
+        ContextualOnboardingViewHighlighter.highlight(view: child, inParent: parent)
+
+        // WHEN
+        let result = ContextualOnboardingViewHighlighter.isViewHighlighted(child)
+
+        // THEN
+        XCTAssertTrue(result)
+    }
+
+    func testWhenIsViewIsNotHighligthedIsCalledAndViewIsHighlightedThenReturnFalse() {
```

Comment:
I think title should be testWhenIsViewHighligthedIsCalledAndViewIsNotHighlightedThenReturnFalse although I am not 100% sure of the meaning f this test

Reply:
copy and paste issue. changed to `testWhenIsViewHighligthedIsCalledAndViewIsNotHighlightedThenReturnFalse`